### ISSUE TITLE
gblast3.py conflict with biopython 1.71

### ISSUE THREAD

**Issue by lucaz88** [SaierLaboratory/BioVx#1]
Hi,
I am not at all a coding expert but I noticed that there is a conflict between the script 'gblast3.py' and the last release of biopython. I noticed that after the upgrade of biopython from the version 1.70 to 1.71 I got the following error just before the HMMTOP step:
```
Loading Substrates
Loading BLAST results
Writing FASTAS
Traceback (most recent call last):
  File "/home/lucaz/BioVx/scripts/gblast3.py", line 717, in <module>
    GB.write_fastas()
  File "/home/lucaz/BioVx/scripts/gblast3.py", line 207, in write_fastas
    SeqIO.write(list(set(myquery)),query_file,'fasta')
TypeError: unhashable type: 'SeqRecord'
```
Cheers,
Luca

**Comment by gaboentropy** [comment]
Thanks Lucaz, we'll look into it.

**Comment by gaboentropy** [comment]
We have fixed the incompatibility with biopython 1.71.
Thanks for letting us know.
